tracing/trace.go

```diff
@@ -65,10 +65,13 @@ func (t *trace) addCounter(key string, delta float64) {
 }
 
 func (t *trace) setTag(key string, value interface{}) {
-	if v, ok := value.(bool); ok {
-		value = strconv.FormatBool(v)
+	var valStr string
+	if v, ok := value.(string); !ok {
+		valStr = fmt.Sprintf("%v", value)
+	} else {
+		valStr = v
 	}
```

Comment:
I'm not sure which is faster, always do `fmt.Sprintf("%v", value)` even if it's string, or do a typecasting test. 😅 Also `ok` being true is slightly different from "`value` is string", as it could be typedef'd string or some other weird cases.

Reply:
I think using `fmt.Sprintf("%v", value)` makes the most sense here :)